genesis/sensors/base_sensor.py

```diff
@@ -64,23 +77,24 @@ class Sensor(RBC):
 
     def __init__(self, sensor_options: "SensorOptions", sensor_idx: int, sensor_manager: "SensorManager"):
         self._options: "SensorOptions" = sensor_options
-        self._idx: int = sensor_idx
+        self._sensor_idx: int = sensor_idx
```

Comment:
Why did you feel the need to rename this attribute? `self._idx` was fine, and more aligned with what we are doing elsewhere.